WordPress/Classes/Models/AbstractPost.m

```diff
@@ -281,6 +282,27 @@ - (Media *)featuredImage
 
 #pragma mark - Convenience methods
 
+- (BOOL)isDraft
+{
+    if ([self.status isEqualToString:PostStatusDraft]) {
```

Comment:
We could. I chose to be more verbose for clarity. I think its easier to read this way.